requirements.txt

```diff
@@ -1,7 +1,4 @@
-paramiko
-python-slugify>=3.0.2
-vcrpy>=2.1.1
-cerberus
-vcrpy-unittest
+# generated from manifests external_dependencies
 boto3
-requests-mock
+paramiko
```

Comment:
do we need these deps at all? They will come from external deps w/ MQT=PIP

Reply:
This is autogenerated so this is not really a problem to mainain IMO

Reply:
FYI I tested removing it and pre-commit made me put it again